packages/maskbook/src/plugins/ITO/hooks/usePoolPayload.ts

```diff
@@ -1,6 +1,28 @@
-import { useAsyncRetry } from 'react-use'
+import { ValueRef } from '@dimensiondev/holoflows-kit'
 import { PluginITO_RPC } from '../messages'
+import { useValueRef } from '../../../utils/hooks/useValueRef'
+import type { JSON_PayloadInMask } from '../types'
+
+const storage = new Map<string, ValueRef<JSON_PayloadInMask>>()
```

Comment:
It seems like you never set on `.value` so you don't need `ValueRef`
```suggestion
const storage = new Map<string, JSON_PayloadInMask>()
```